### PR TITLE
Improvements for version 0.9.0

### PR DESCRIPTION
**Pull request by thomasmorgan**
a number of edits mostly fixing recent issues. See commits below.

This probably will break current experiments (though examples have been updated), so we should use versioning to merge.

We should also update the wiki after merging to reflect the changes

### CODE REVIEW

File: wallace/command_line.py [hunk 1]
```diff
@@ -344,6 +344,11 @@ def deploy_sandbox_shared_setup(verbose=True, app=None, web_procs=1):
 
     database_size = config.get('Database Parameters', 'database_size')
 
+    try:
+        whimsical = config.get('Easter eggs', 'whimsical') == "true"
```
Comment (suchow) [inline wallace/command_line.py]: Instead of `=== "true"`, ConfigParser has a function `getboolean` which does the conversion for you.

File: wallace/command_line.py [hunk 2]
```diff
@@ -344,6 +344,11 @@ def deploy_sandbox_shared_setup(verbose=True, app=None, web_procs=1):
...
+    except:
+        whimsical = "false"
```
Comment (suchow) [inline wallace/command_line.py]: (If you make the above change, this then becomes `whimsical = False`.)

File: wallace/heroku/clock.py [hunk 3]
```diff
@@ -153,44 +165,57 @@ def check_db_for_missing_notifications():
...
+ Wallace. You are receiving this email because the Wallace platform has
+ discovered evidence that a notification from Amazon Web Services failed to
+ arrive at the server. Wallace has automatically contacted AWS and has
+ determined the dropped notification was an abandoned/returned notification
+ (i.e. the participant had returned the experiment or had run out of time).
+ This is a serious error and so Wallace has paused the experiment - expiring
+ the HIT on MTurk and setting auto_recruit to false. Participants currently
+ playing will be able to finish, however no further participants will be
+ recruited until you do so manually. We strongly suggest you use the details
+ below to check the database to make sure the missing notification has caused
```
Comment (suchow) [inline wallace/heroku/clock.py]: This should be "has not".

File: examples/bartlett1932/experiment.py [hunk 4]
```diff
@@ -33,6 +32,10 @@ def setup(self):
             for net in self.networks():
                 WarOfTheGhostsSource(network=net)
 
+    def new_network(self):
```
Comment (suchow) [inline examples/bartlett1932/experiment.py]: I think we should call this `create_network()`.

File: wallace/command_line.py [hunk 5]
```diff
@@ -844,4 +844,54 @@ def verify_package(verbose=True):
         log("✓ README is OK",
             delay=0, chevrons=False, verbose=verbose)
 
+    # Check directories.
+    if not os.path.exists("static/scripts"):
```
Comment (suchow) [inline wallace/command_line.py]: Why is this required?

File: wallace/command_line.py [hunk 6]
```diff
@@ -844,4 +844,54 @@ def verify_package(verbose=True):
...
+        log("✗ static/scripts directory is MISSING",
+            delay=0, chevrons=False, verbose=verbose)
+        return False
+    elif not os.path.exists("templates"):
+        log("✗ templates directory is MISSING",
+            delay=0, chevrons=False, verbose=verbose)
+        return False
+    elif not os.path.exists("static/css"):
```
Comment (suchow) [inline wallace/command_line.py]: Why is this required?

**Comment by thomasmorgan** [comment]
Jordan - don't merge this yet. I'm going to put the tests on it.
